lisa/transformers/dom0_kernel_installer.py

```diff
@@ -124,10 +127,19 @@ def install(self) -> str:
                 node.get_pure_path(f"/boot/initrd.img-{new_kernel}"),
             )
         else:
+            # Mariner 2.0 initrd
+            target = f"/boot/initrd.img-{current_kernel}"
+            link = f"/boot/initrd.img-{new_kernel}"
+
+            if distro == "Microsoft Azure Linux 3.0":
```

Comment:
Check it by `isinstance(node.os, Mariner)` and the `node.os.information.version`. Not use the string.

Reply:
Sure Chi

Reply:
Done